gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/handler/MultiTaskSession.java

```diff
@@ -126,6 +140,8 @@ public boolean submitTask(final SessionTask sessionTask) throws RejectedExecutio
 
     @Override
     public void run() {
+        sessionTaskStarted.set(true);
```

Comment:
I see some duplicate logic in general throughout the changed Session types.

Is it possible to put this stuff in the `AbstractSession` and invoke super?